### PR TITLE
Prepare 0.0.60 release to support React Native 0.60.0 and above

### PR DESCRIPTION
**Pull request by davidwico**
**Changes**
* replace NestedScrollView with ScrollView (no issues on react-native 0.60.0+), remove react-native-nested-scroll-view dependency
* remove react-native-iphone-x-helper dependency (no need dependency for one util function)
* remove all tests and their dependencies like: enzyme, jest etc.
* remove unused scripts from package.json
* remove jest config from package.json
* add react and react-native as peerDependencies
* remove patches
* add react-native.config.js file to enable font linking
* add .DS_Store to .gitignore
* add prettier to project, to avoid developers private style settings overriding current one (there is also space to configure it better)
* add prettier to eslint config
* update license year
* update README installation guide for RN >= 0.60.0
* remove babel.config.js, I think it's not needed here because project is not runnable
* replace package-lock.json with yarn.lock
* update library version in package.json
* update all libraries from package.json

**About linking**

There is need to run command
`react-native link react-native-sticky-parallax-header`
in order to link custom fonts
Auto-linking must be done later, proper topic is open as issue on react-native-community/cli Github page

### CODE REVIEW

File: src/components/ScrollableTabBar/ScrollableTabBar.js [hunk 1]
```diff
@@ -113,7 +112,7 @@ class ScrollableTabBar extends React.PureComponent {
           }
         ]}
       >
-        <NestedScrollView
+        <ScrollView
```
Comment (Karniej) [inline src/components/ScrollableTabBar/ScrollableTabBar.js]: Have you checked if this solution works on Android correctly ?

File: src/predefinedComponents/components/UserModal/UserModal.styles.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import { StyleSheet, Platform } from 'react-native'
-import { ifIphoneX } from 'react-native-iphone-x-helper'
+import { ifIphoneX } from '../../../utils'
```
Comment (Karniej) [inline src/predefinedComponents/components/UserModal/UserModal.styles.js]: Good job on the removing the dependency 👍

File: src/components/ScrollableTabBar/ScrollableTabBar.js (hunk 1 above)
Comment (davidwico) [reply src/components/ScrollableTabBar/ScrollableTabBar.js]: Yes, it works good on versions 0.60.0+
I think ScrollView was fixed by React Native

File: src/predefinedComponents/components/UserModal/UserModal.styles.js (hunk 2 above)
Comment (davidwico) [reply src/predefinedComponents/components/UserModal/UserModal.styles.js]: Thanks

**Comment by masiamj** [comment]
@Karniej Thoughts about merging this in soon?

**Pull request merged**